.golangci.yml

```diff
@@ -0,0 +1,40 @@
+run:
+  timeout: 5m
+linters:
+  enable:
+    # Simple linter to check that your code does not contain non-ASCII identifiers.
+    - asciicheck
+    # Go linter that checks if package imports are in a list of acceptable packages.
+    #- depguard
+    # Checks for pointers to enclosing loop variables.
+    - exportloopref
+    # Gofmt checks whether code was gofmt-ed
+    - gofmt
+    # Inspects source code for security problems.	
+    - gosec
+    # Finds naked returns in functions greater than a specified function length.
+    - nakedret
+    # Reports ill-formed or insufficient nolint directives.
+    - nolintlint
+    # Checks for dangerous unicode character sequences.
+    - bidichk
+  disable:
+    # ignore unused check
+    - unused
```

Comment:
If go vet is executed, unused functions or variables will no longer appear here. Enable it or delete it directly.

Reply:
> If go vet is executed, unused functions or variables will no longer appear here. Enable it or delete it directly.

Are you suggesting that I remove the 'unused' configuration and remove unused variables and functions in this project code?

Reply:
yes, but you seems to have deleted them in code?

Reply:
> yes, but you seems to have deleted them in code?

o, this rule 'unused' considers fields in a struct and functions belonging to a struct as well. So, I'm unsure if I should remove them, as I don't know if they are useful.

Reply:
> yes, but you seems to have deleted them in code?

no, this rule 'unused' considers fields in a struct and functions belonging to a struct as well. So, I'm unsure if I should remove them, as I don't know if they are useful.

Reply:
> yes, but you seems to have deleted them in code?

no, this rule 'unused' considers fields in a struct and functions belonging to a struct as well. So, I'm unsure if I should remove them, as I don't know if they are useful in the future.

Reply:
I think they can be removed.